src/vs/platform/environment/node/userDataPath.d.ts

```diff
@@ -3,7 +3,7 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { NativeParsedArgs } from 'vs/platform/environment/common/argv';
+import { NativeParsedArgs } from '../../../platform/environment/common/argv.js';
```

Comment:
Do we need this change? seems too work as is in upstream repo?

Reply:
Yep, we don't need it, update because I tried to global search `from 'vs` to see what's missing